instrumentation/README.md

```diff
@@ -20,22 +20,39 @@ Once instrumented, Java executables send traces to a locally running
 
 ### Environment Variables
 
-The way the .so causes a Java executable to be instrumented is simply by setting two environment variables,
-JAVA_TOOL_OPTIONS and OTEL_SERVICE_NAME.
+The way the .so causes a Java executable to be instrumented is by setting the environment variables, JAVA_TOOL_OPTIONS
+and OTEL_SERVICE_NAME, and optionally, OTEL_EXPORTER_OTLP_ENDPOINT and OTEL_RESOURCE_ATTRIBUTES.
 
 #### JAVA_TOOL_OPTIONS
 
 This environment variable contains a `-javaagent` flag set to the full path of the Splunk OTel Java JAR.
 
 e.g. `JAVA_TOOL_OPTIONS='-javaagent:/usr/lib/splunk-instrumentation/splunk-otel-javaagent.jar'`
 
+This variable is populated by the .so by concatenating the `java_agent_jar` attribute the config to a `-javaagent:` prefix.
+
 #### OTEL_SERVICE_NAME
 
 This environment variable sets the service name for the soon-to-be running Java application, usually derived from the
 arguments of the Java executable (can be overridden via the config).
 
+This variable is set directly from the `service_name` attribute in the config.
+
 _Meta: link to docs about how service name is used and why it's required._
 
+#### OTEL_EXPORTER_OTLP_ENDPOINT
```

Comment:
Since we are providing this feature with the collector and the customer will go through the collector wizard, we need not provide the option of overriding the endpoint. The traces should be sent to the local collector endpoint itself, users cannot provide the ingest url for traces to be sent directly to the backend if they want zc autoinstrumentation.

Reply:
Ok -- will delete.